mojo/public/cpp/platform/tcp_platform_handle_utils_posix.cc

```diff
@@ -120,7 +120,18 @@ base::ScopedFD CreateTCPServerHandle(size_t port) {
     PLOG(ERROR) << "listen" << handle.get();
     return base::ScopedFD();
   }
-  LOG(INFO) << "Server Sock fd for port " << port <<":"<<handle.get();
+
+  // Get port number
+  if (port == 0) {
+    struct sockaddr_in sin;
+    socklen_t len = sizeof(sin);
+    if (getsockname(handle.get(), (struct sockaddr*)&sin, &len) < 0)
+      PLOG(ERROR) << "getsockname() " << handle.get();
+    else
+      port = *out_port = ntohs(sin.sin_port);
```

Comment:
How about adding DCHECK(out_port) in this case?

Reply:
Done. I added CHECK().